autoware_launch/config/planning/scenario_planning/lane_driving/motion_planning/obstacle_cruise_planner/obstacle_cruise_planner.param.yaml

```diff
@@ -89,7 +89,7 @@
 
       # hysteresis for cruise and stop
       obstacle_velocity_threshold_from_cruise_to_stop : 3.0 # stop planning is executed to the obstacle whose velocity is less than this value [m/s]
-      obstacle_velocity_threshold_from_stop_to_cruise : 3.5 # stop planning is executed to the obstacle whose velocity is less than this value [m/s]
+      obstacle_velocity_threshold_from_stop_to_cruise : 2.5 # stop planning is executed to the obstacle whose velocity is less than this value [m/s]
```

Comment:
`obstacle_velocity_threshold_from_stop_to_cruise` has to be larger than `obstacle_velocity_threshold_from_cruise_to_stop`.
I guess when the ego's velocity is 2.8, the mode will chatter like cruise -> stop -> cruise -> stop -> ...

Reply:
Thanks @takayuki5168 -san for your review comment.
I have discussed this comment with @brkay54 as he had previous knowledge in this point and we come to the following updates in the parameters .
- We see that having `obstacle_velocity_threshold_from_stop_to_cruise` with 1.5 m/s (5.4 k/h) would be better handling this scenario allowing the ego vehicle to cruise the VRU not always triggering obstacle stop imitating natural human driving in such dense urban cases
- As well having `obstacle_velocity_threshold_from_cruise_to_stop` less than `obstacle_velocity_threshold_from_stop_to_cruise` will prevent chattering.

I have provided new demo videos under **After this PR**. Please have a look and let us know your thoughts.

Reply:
@ahmeddesokyebrahim 
Thank you for the explanation.

In the use case where the ego tries to stop for the front stopped object, decreasing the parameter makes the ego late to switch from CRUISE to STOP, and as a result, the ego is late to decelerate which we feel is very dangerous. So I don't agree with decreasing this parameter.
Does this explanation make sense to you?

Reply:
Thanks @takayuki5168 -san for your reply.

> decreasing the parameter makes the ego late to switch from CRUISE to STOP

Actually it is intended by this change to let the ego vehicle cruise the front object not to trigger obstacle stop for it, trying to have a natural human driving behavior in such dense urban areas.

Do you have in mind or with prior experiments any minimum threshold that we can have for `obstacle_velocity_threshold_from_cruise_to_stop` & `obstacle_velocity_threshold_from_stop_to_cruise` ?